user-interface/frontend/themes/datamanager/components/main.css

```
@@ -81,11 +81,19 @@
 }
 
 .main.legal-notice {
-  padding: var(--lumo-space-m);
+  padding: var(--lumo-space-xl);
+  height: fit-content;
+  max-width: 60rem;
+  display: flex;
+  margin: auto;
 }
 
 .main.data-privacy-agreement {
-  padding: var(--lumo-space-m);
+  padding: var(--lumo-space-xl);
+  height: fit-content;
+  max-width: 60rem;
```

I recommend to replace the max-width property with the with attribute employing the [clamp ](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp)functionality. 
This ensures that a minimum, preferred(ideally dependent on the viewport width) and maximum value is set. 
```suggestion
width: clamp(500px, 60vw, 100%);
```